### DISCUSSION TITLE
POA&M generation questions (Also a question about 7.0.0.0alpha1)

### DISCUSSION THREAD

**Discussion by dante-asriel-price** [Vulnerator/Vulnerator#231] [Q&A]
Hey everyone!

I had a few questions for the community on using and modifying Vulnerator. I'm currently using version 6.1.9 for generating POA&Ms using Fortify data and a STIG checklist, working in a Software Assurance capacity, so I'll ask that question first: Is there a way to use an updated POA&M template in v6.1.9? I have one, but cannot for the life of me find where Vulnerator is keeping its template. 

My second question is about v7.0.0.0alpha1: Is fortify data going to be put in the POA&Ms in the production release? It currently processes the file but does not include anything in the generated POA&M. 

Thank you!

**Comment by lifeincha0s** [comment]
POA&M templates are used. A custom excel file is formatted and populated fot each execution. Check out OpenXmlReportCreator.cs in the Model folder.
As for using 7.0, I ran into a lot of bugs I had to fix in order to get it to compile. I didn't go any further than that since 6.1.9 already worked. I have since added several feature updates using the 6.1.9 code base.
